He_rel_Coulomb_Gaunt_JK.py

```diff
@@ -54,15 +55,12 @@
 ################### Define V potential ######################
 def u(r):
     u = erf(r)/r + (1/(3*np.sqrt(np.pi)))*(np.exp(-(r**2)) + 16*np.exp(-4*r**2))
-    #erf(r) is an error function that is supposed to stop the potential well from going to inf.
-    #if i remember correctly
     return u
 
 def V(x):
```

Comment:
Same here. Check the one in `NuclearPotential.py`. It takes also the nuclear charge, the precision and the origin as arguments